src/main/java/com/google/cloud/tools/app/Option.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.tools.app;
+
+/**
+ * Contains a flag short ("-f") and long ("--force") forms, along with their type.
+ */
+public enum Option {
+  ENABLE_QUICKSTART("enable_quickstart", Type.BOOLEAN),
```

Comment:
I believe this is a beta option for app developers. The user generally configures this.